psykorebase/cli.py

```diff
@@ -166,7 +166,7 @@ def psykorebase(args=sys.argv[1:]):
     rebaser = factory(repository_path,
                       primary_branch,
                       secondary_branch)
-    print "Working with {dvcs}, rebasing {secondary} on top of {primary}" \
+    print("Working with {dvcs}, rebasing {secondary} on top of {primary}"
```

Comment:
Same remarks here. print a tuple with only 1 item is the same as printing the item